pkg/cli/nop_ui.go

```diff
@@ -0,0 +1,12 @@
+package cli
+
+var NopUI = &nopUIImpl{}
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported var NopUI should have comment or be unexported